.dockerignore

```diff
@@ -5,3 +5,5 @@ __pycache__
 .vagrant
 Vagrantfile
 .cache
+webapp-django/node_modules
+webapp-django/static
```

Comment:
Why add these to `.dockerignore`? It means they don't get copied into the image with a COPY.

Reply:
We don't want `webapp-django/node_modules` because the node dependencies are currently being installed in `webapp-frontend-deps`. I have a `node_modules` in my webapp directory because that's how Atom finds which eslint to run for linting the project, and having it get copied in masks problems with missing dependencies that come up during automation, when there is no `node_modules` to copy.

`static` is in there because it gets built during image creation and doesn't need to be copied in. `.dockerignore` doesn't affect volume mounts so the static files still get correctly updated during development.